chapters/features.txt

```diff
@@ -7596,24 +7598,31 @@ following table:
 
 [[external-memory-handle-types-compatibility]]
 .External memory handle types compatibility
+[width="100%",options="header",cols="3*"]
 |====
-| Handle type | sname:VkPhysicalDeviceIDProperties::pname:driverUUID | sname:VkPhysicalDeviceIDProperties::pname:deviceUUID
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_FD_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_OPAQUE_WIN32_KMT_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_D3D11_TEXTURE_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_D3D11_TEXTURE_KMT_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_D3D12_HEAP_BIT | Must match | Must match
-| ename:VK_EXTERNAL_MEMORY_HANDLE_TYPE_D3D12_RESOURCE_BIT | Must match | Must match
+| Handle Type +
+  etext:VK_EXTERNAL_MEMORY_HANDLE_TYPE_*
+
+| sname:VkPhysicalDeviceIDProperties::{zwsp}pname:driverUUID
+| sname:VkPhysicalDeviceIDProperties::{zwsp}pname:deviceUUID
+
+
+| etext:OPAQUE_FD_BIT | Must match | Must match
```

Comment:
I appreciate what you're trying to do, but replacing enames with etext shorthand makes me nervous (and violates the style guide). It can't be validated, and searching for the full enumerant name won't find it.  Yes, some of the names are very long, but we'd like to find another way to address the problem.

Reply:
Ok. It should end up in `<code>`, so I may attempt a CSS solution.
The PDF does something like that and line-breaks even withing the keyword, if it has to.

I was not happy with it either for searchability sake. `*text` macros though seem for exactly this purpose &mdash; formating as `*name` things that are for various reasons not Vulkan entities (e.g. wildcarded names).